.github/workflows/ElastixGitHubActions.yml

```diff
@@ -67,11 +68,15 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install ninja
         
-    
+    - name: Get specific version of CMake, Ninja
+      uses: lukka/get-cmake@v3.18.3
+      
     - name: Download ITK
       if: steps.cache.outputs.cache-hit != 'true'
       run: |
-        git clone https://github.com/InsightSoftwareConsortium/ITK.git --branch ${{ matrix.itk-git-tag }} --depth 1 ITK-source
+        git clone https://github.com/InsightSoftwareConsortium/ITK.git ITK-source
+        cd ITK-source
+        git checkout ${{ matrix.itk-git-tag }}
```

Comment:
Just wondering, did git argument  `--branch ${{ matrix.itk-git-tag }}` not work well? `--depth 1` might speed things up, but I'm not sure if it does, honestly.

Reply:
This doesn't work if you want to checkout a specific commit. (Both the --branch flag and the --depth flag)
Apparently git needs more depth to checkout commits. ;)